docs/content/guide/component-router.ngdoc

```diff
@@ -6,18 +6,17 @@
 # Component Router
 
 <div class="alert alert-info">
-**Deprecation Notice:** In an effort to keep synchronized with router changes in Angular 2, this implementation of the Component Router (ngComponentRouter module) has been deprecated and will not receive further updates.
-We are investigating backporting the Angular 2 Router to Angular 1, but alternatively, use the {@link ngRoute} module or community developed projects (e.g. [ui-router](https://github.com/angular-ui/ui-router)).
+**Deprecation Notice:** In an effort to keep synchronized with router changes in Angular, this implementation of the Component Router (ngComponentRouter module) has been deprecated and will not receive further updates.
+We are investigating backporting the Angular Router to AngularJS, but alternatively, use the {@link ngRoute} module or community developed projects (e.g. [ui-router](https://github.com/angular-ui/ui-router)).
```

Comment:
I feel this is a little confusing (Angular and AngularJS on the same sentence). What is the guideline for places were Angular and AngularJS are used side-by-side? I think a clarification (e.g. at the top of the page) would be nice.